h5py/utils.pyx

```diff
@@ -24,10 +24,13 @@ cdef inline void* emalloc(size_t size) except? NULL:
     """Wrapper for malloc(size) with the following behavior:
 
     1. Always returns NULL for emalloc(0)
-    2. Raises MemoryError for emalloc(size<0) and returns NULL
+    2. Raises MemoryError for emalloc(size<0) and returns NULL **IMPOSSIBLE as size>=0** 
```

Comment:
I agree. Let's remove this line from the docstring.

Reply:
Done